jetbrains-core/build.gradle

```diff
@@ -1,8 +1,20 @@
 // Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+import software.aws.toolkits.telemetry.generator.gradle.GenerateTelemetry
+
 apply plugin: 'org.jetbrains.intellij'
 
+buildscript {
+    repositories {
+        mavenCentral()
+        maven { setUrl("https://jitpack.io") }
+    }
+    dependencies {
+        classpath("software.aws.toolkits:telemetry-generator:0.0.14")
```

Comment:
should we put this stuff in the `telemetry-client` module?

Reply:
I was trying to go in the direction of getting rid of that module, we can move it to shared and reuse it in other products that are still using Pinpoint